Cargo.toml

```diff
@@ -11,8 +11,9 @@ keywords = ["cargo", "workspace", "lint", "diagnostics"]
 readme = "./README.md"
 
 [dependencies]
-cargo_metadata = "0.18.1"
+cargo_metadata = "0.19.0"
 clap = { version = "4.4.11", features = ["derive"] }
+serde_json = "1"
```

Comment:
Is there a reason why you added a direct `serde_json` dependency? As best I can tell, everything still works without the extra dependency

Reply:
Let me recheck

Might have been an artifact of some debugging session 

Will update.

No rush on review

Enjoy the holidays

Reply:
Cleaned it up